erpnext/patches/v13_0/rename_ksa_qr_field.py

```diff
@@ -12,5 +13,20 @@ def execute():
 
 	if frappe.db.exists('DocType', 'Sales Invoice'):
 		frappe.reload_doc('accounts', 'doctype', 'sales_invoice', force=True)
+
+		# rename_field method assumes that the field already exists or the doc is synced
+		if not frappe.db.has_column('Sales Invoice', 'ksa_einv_qr'):
```

Comment:
Shouldn't [this](https://github.com/frappe/erpnext/blob/develop/erpnext/regional/saudi_arabia/utils.py#L19-L30) be enough? :thinking:

Reply:
Then there is no point of the patch then, the patch will get executed before the field is created and not data will me moved from the old field in the invoices